### PR TITLE
make fmt

### PR DESCRIPTION
**Pull request by johnbelamaric**

### CODE REVIEW

**Comment by johnbelamaric** [comment]
/hold

Still working on some things; going to run 008.sh and 009.sh and make sure there are no more tweaks needed.

File: e2e/lib/k8s.sh [hunk 1]
```diff
@@ -126,41 +126,80 @@ function k8s_exec {
     kubectl --kubeconfig $kubeconfig -n $resource_namespace exec $resource_name -- /bin/bash -c "$command"
     return $?
 }
+
+function _k8s_quantity_factor {
```
Comment (electrocucaracha) [inline e2e/lib/k8s.sh:130]: A R1+ improvement here will be using `numfmt` tool

**Comment by johnbelamaric** [comment]
/hold cancel

Ok, this is good to go.

**Pull request merged**